crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -907,13 +907,50 @@ impl Command for LogComponents {
     }
 }
 
+/// Trait to execute some [`Commands`] on some [`World`].
+pub trait Execute {
+    fn execute<F: FnOnce(&World, Commands) -> R, R>(self, f: F) -> R;
+}
+
+impl Execute for &mut World {
+    /// Creates a new [`Commands`] instances and passes it to the given function, then executes it on this [`World`].
+    ///
+    /// # Usage
+    ///
+    /// This is not an efficient method of executing commands. However, it can be a convenient tool for
```

Comment:
This should explain why it's not efficient, and the more efficient alternatives.